toolkit/tools/imagegen/configuration/configuration_test.go

```diff
@@ -531,7 +529,6 @@ var expectedConfiguration Config = Config{
 			},
 			KernelOptions: map[string]string{
 				"default": "kernel",
-				"hyperv":  "kernel-hyperv",
```

Comment:
I think we could do a similar update inside `imagegen/attendedinstaller/_manualrun/manualrun.go`.

Reply:
manualrun.go does not contain hyperv options. This file is used as a tool to quickly test the iso installer flow and the package lists are not explicitly checked. However, they may be used as inputs to other tools so I will fix while I am here.